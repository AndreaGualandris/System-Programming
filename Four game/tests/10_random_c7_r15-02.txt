2
0
5
5
6
3
1
2
1
1
0
5
0
6
6
6
1
5
1
2
3
4
3
2
1
6
3
5
1
4
5
6
2
3
3
6
3
6
6
5
3
5
6
4
6
6
3
0
2
0
5
5
1
5
6
2
0
1
1
5
0
4
6
0
2
5
5
1
6
6
1
4
5
6
1
3
0
6
6
3
0
5
1
4
1
5
5
5
6
0
5
5
6
1
3
6
6
5
4
2
5
1
1
2
4
