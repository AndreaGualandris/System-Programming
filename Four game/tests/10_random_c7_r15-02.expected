-------
       
       
       
       
       
       
       
       
       
       
       
       
       
       
       
-------
0123456
Player 'o' to move:
  o
  v
-------
       
       
       
       
       
       
       
       
       
       
       
       
       
       
  o    
-------
0123456
Player 'x' to move:
x
v
-------
       
       
       
       
       
       
       
       
       
       
       
       
       
       
x o    
-------
0123456
Player 'o' to move:
     o
     v
-------
       
       
       
       
       
       
       
       
       
       
       
       
       
       
x o  o 
-------
0123456
Player 'x' to move:
     x
     v
-------
       
       
       
       
       
       
       
       
       
       
       
       
       
     x 
x o  o 
-------
0123456
Player 'o' to move:
      o
      v
-------
       
       
       
       
       
       
       
       
       
       
       
       
       
     x 
x o  oo
-------
0123456
Player 'x' to move:
   x
   v
-------
       
       
       
       
       
       
       
       
       
       
       
       
       
     x 
x ox oo
-------
0123456
Player 'o' to move:
 o
 v
-------
       
       
       
       
       
       
       
       
       
       
       
       
       
     x 
xoox oo
-------
0123456
Player 'x' to move:
  x
  v
-------
       
       
       
       
       
       
       
       
       
       
       
       
       
  x  x 
xoox oo
-------
0123456
Player 'o' to move:
 o
 v
-------
       
       
       
       
       
       
       
       
       
       
       
       
       
 ox  x 
xoox oo
-------
0123456
Player 'x' to move:
 x
 v
-------
       
       
       
       
       
       
       
       
       
       
       
       
 x     
 ox  x 
xoox oo
-------
0123456
Player 'o' to move:
o
v
-------
       
       
       
       
       
       
       
       
       
       
       
       
 x     
oox  x 
xoox oo
-------
0123456
Player 'x' to move:
     x
     v
-------
       
       
       
       
       
       
       
       
       
       
       
       
 x   x 
oox  x 
xoox oo
-------
0123456
Player 'o' to move:
o
v
-------
       
       
       
       
       
       
       
       
       
       
       
       
ox   x 
oox  x 
xoox oo
-------
0123456
Player 'x' to move:
      x
      v
-------
       
       
       
       
       
       
       
       
       
       
       
       
ox   x 
oox  xx
xoox oo
-------
0123456
Player 'o' to move:
      o
      v
-------
       
       
       
       
       
       
       
       
       
       
       
       
ox   xo
oox  xx
xoox oo
-------
0123456
Player 'x' to move:
      x
      v
-------
       
       
       
       
       
       
       
       
       
       
       
      x
ox   xo
oox  xx
xoox oo
-------
0123456
Player 'o' to move:
 o
 v
-------
       
       
       
       
       
       
       
       
       
       
       
 o    x
ox   xo
oox  xx
xoox oo
-------
0123456
Player 'x' to move:
     x
     v
-------
       
       
       
       
       
       
       
       
       
       
       
 o   xx
ox   xo
oox  xx
xoox oo
-------
0123456
Player 'o' to move:
 o
 v
-------
       
       
       
       
       
       
       
       
       
       
 o     
 o   xx
ox   xo
oox  xx
xoox oo
-------
0123456
Player 'x' to move:
  x
  v
-------
       
       
       
       
       
       
       
       
       
       
 o     
 o   xx
oxx  xo
oox  xx
xoox oo
-------
0123456
Player 'o' to move:
   o
   v
-------
       
       
       
       
       
       
       
       
       
       
 o     
 o   xx
oxx  xo
ooxo xx
xoox oo
-------
0123456
Player 'x' to move:
    x
    v
-------
       
       
       
       
       
       
       
       
       
       
 o     
 o   xx
oxx  xo
ooxo xx
xooxxoo
-------
0123456
Player 'o' to move:
   o
   v
-------
       
       
       
       
       
       
       
       
       
       
 o     
 o   xx
oxxo xo
ooxo xx
xooxxoo
-------
0123456
Player 'x' to move:
  x
  v
-------
       
       
       
       
       
       
       
       
       
       
 o     
 ox  xx
oxxo xo
ooxo xx
xooxxoo
-------
0123456
Player 'o' to move:
 o
 v
-------
       
       
       
       
       
       
       
       
       
 o     
 o     
 ox  xx
oxxo xo
ooxo xx
xooxxoo
-------
0123456
Player 'x' to move:
      x
      v
-------
       
       
       
       
       
       
       
       
       
 o     
 o    x
 ox  xx
oxxo xo
ooxo xx
xooxxoo
-------
0123456
Player 'o' to move:
   o
   v
-------
       
       
       
       
       
       
       
       
       
 o     
 o    x
 oxo xx
oxxo xo
ooxo xx
xooxxoo
-------
0123456
Player 'x' to move:
     x
     v
-------
       
       
       
       
       
       
       
       
       
 o     
 o   xx
 oxo xx
oxxo xo
ooxo xx
xooxxoo
-------
0123456
Player 'x' wins!
